charts/privatebin/values.yaml

```
@@ -44,7 +44,9 @@ controller:
     ## https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
     ##
     # storageClass: "-"
-
+  # For deployment controller, an emptyDir can be enable in the pod for writing data
+  # with the following option
+  emptyDir: true
```

```suggestion
  emptyDir: false
```

I would prefer to leave this `false` as default, since that's current behavior, unless there is a compelling reason to change it.